setup.py

```diff
@@ -65,103 +77,91 @@ def run(self):
             sys.exit(info)
 
 
+class PostDevelopCommand(develop):
+    """
+    Post-installation for development mode.
+    Execute manually with python setup.py develop or automatically included with
+    `pip install -e . -r dev-requirements.txt`.
+    """
+    def run(self):
+
+        # super
+        develop.run(self)
+
+        # development setup scripts (pre-requirements)
+        subprocess.call(f"scripts/installation/install_solc.py")
+        subprocess.call(".circleci/install_circle_cli.sh")
+
 #
-#  Dependencies
+#  Requirements
 #
 
-with open(os.path.join(BASE_DIR, "requirements.txt")) as f:
-    _PIP_FLAGS, *INSTALL_REQUIRES = f.read().split('\n')
 
+def read_requirements(path):
+    with open(os.path.join(BASE_DIR, path)) as f:
+        _PIPENV_FLAGS, *REQUIREMENTS = f.read().split('\n')
+    return REQUIREMENTS
```

Comment:
Since you abstracted this as a function, using caps for these variables seems weird.

Reply:
Agreed - I will make an update.

Reply:
:heavy_check_mark: